python/console/console.py

```diff
@@ -96,6 +135,33 @@ def closeEvent(self, event):
         QWidget.closeEvent(self, event)
 
 
+class PythonConsoleW(QWidget):
```

Comment:
Better write this completely (`PythonConsoleWidget` or whatever it's meant to be, so we do not mistake it with `PythonConsoleWinterfell`, `PythonConsoleWhistleblower` or `PythonConsoleWindow`)